httpd/handler_test.go

```diff
@@ -87,13 +87,13 @@ func TestBatchWrite_UnmarshalEpoch(t *testing.T) {
 		t.Logf("testing %q\n", test.name)
```

Comment:
Let me add that in a separate PR -- I will note it down now, as I have an idea about this testing that is larger in scope.